helm/templates/configmap.yaml

```diff
@@ -7,12 +7,32 @@ metadata:
 data:
   application.conf: |-
     document.store {
-      dataStoreType = mongo
+      dataStoreType = {{ .Values.configMap.dataStoreType }}
+      {{- if eq .Values.configMap.dataStoreType "mongo" }}
       mongo {
         {{- if .Values.configMap.mongodb.url }}
         url = {{ .Values.configMap.mongodb.url | quote }}
         {{- else }}
         host = {{ .Values.configMap.mongodb.host }}
         {{- end }}
       }
+      {{ else if eq .Values.configMap.dataStoreType "postgres" }}
+      postgres {
+        {{- if .Values.configMap.postgres.url }}
+        url={{ .Values.configMap.postgres.url | quote }}
+        {{- else }}
+        host={{ .Values.configMap.postgres.host }}
+        port={{ .Values.configMap.postgres.port }}
+        {{- end }}
+        {{- if .Values.configMap.postgres.database }}
+        database= {{ .Values.configMap.postgres.database }}
+        {{- end }}
+        {{- if .Values.configMap.postgres.user }}
```

Comment:
we should combine check for both username and password.

Reply:
Addressed.